### ISSUE TITLE
having trouble with OBS Mac 0.11.2 and multiple browser sources for twitchalerts.com

### ISSUE THREAD

**Issue by grubbins2** [obsproject/obs-browser#8]
https://obsproject.com/mantis/view.php?id=286

I have the follow/donate/host alerts working just fine with browser source on the Mac 0.11.2. However, when I try to add the donation goal or recent donations widgets, they always just display as empty "black" boxes. I know the URLs work because I can open them in other browsers just fine. Any idea what is going on or how to solve it? I am not seeing any errors in the logs related to CEF at all. Thanks!
1. Remove all of jobs-studio settings folder.
2. Reinstall OBS 0.11.2 from home page.
3. Reinstall Browser Plugin 1.0.10 from https://obsproject.com/forum/resources/browser-plugin.115/ 
4. Add a new browser source and change URL to: http://www.twitchalerts.com/widgets/donation-goal?token=65C6E84B2D3011756789
5. I only see a blank black box where I should see a donation goal widget.

Additional Information  OS X 10.10.5 (14F27)
MacBook Pro (Retina, 15-inch, Mid 2015)
AMD Radeon R9 M370X 2048 MB

**Comment by grubbins2** [comment]
FYI - I have also tried 32-bit 1.0.6 and received the same results

**Comment by grubbins2** [comment]
From the OBS ticket:
>  This isn't working because no Access-Control-Allow-Origin header is being sent by Twitch alerts, 
> resulting in CEF blocking the request being done by the twitch alerts javascript. This happens because 
> the cef web-security setting is enabled.

**Comment by grubbins2** [comment]
From twitchalerts forums:
> FYI - started a related thread on twitchalerts.com support:
> http://community.twitchalerts.com/t/donation-goal-and-donation-ticker-not-working-in-cef/8790

> JAMACANBACN TwitchAlerts Support 1d
> We can take a look into what the obs team said but if it works in the popout window then the
> problem is probably related to a issue with the browser source and it may also be something
> conflicted with mac as well.

**Comment by grubbins2** [comment]
FYI from IRC chat:
```
[18:42] <grubbins2> hello OBS chat!
[18:43] <grubbins2> having trouble with OBS Mac 0.11.2 and multiple browser sources for twitchalerts.com
[18:43] <c3r1c3_win> grubbins2: What kind of troubles?
[18:44] <grubbins2> I have one URL working just fine for follows/donates/hosts: http://www.twitchalerts.com/alert-box/v3/65C6E84B2D3011756789 [^]
[18:44] <grubbins2> but when I try to add a 2nd URL the box always remains completely black/blank: http://www.twitchalerts.com/widgets/donation-goal?token=65C6E84B2D3011756789 [^]
[18:45] <grubbins2> or if I try adding this donation ticker I also get the same black/blank box: http://www.twitchalerts.com/widgets/donation-ticker?token=65C6E84B2D3011756789 [^]
[18:45] <c3r1c3_win> grubbins2: Let me try and se if I can replicate the issue.
[18:45] <grubbins2> that would be great - thank you
[18:46] <c3r1c3_win> grubbins2: Okay... so the last likn you posted works by itself
[18:46] <grubbins2> yes I have tested all the links and they seem to work in a regular browser
[18:47] <c3r1c3_win> No... I'm testing in OBS
[18:49] <c3r1c3_win> grubbins2: Hmm... the 2nd link doesn't work when the 3rd link is in there
[18:50] <c3r1c3_win> grubbins2: Interesting....
[18:50] <grubbins2> yes c3r1c3_win that is what I am seeing too - the first one works, but the 2nd two do not
[18:50] <c3r1c3_win> grubbins2: When I disable link3, nothing changes, but if I remove the source from the scene, link 2 magically showsup
[18:52] <c3r1c3_win> grubbins2: Okay... so I now have link2 and 3 working at the same time...
[18:52] <grubbins2> oh really?
[18:53] <c3r1c3_win> grubbins2: What's the resolution of Browser Source that you added?
[18:53] <grubbins2> link1: 800x600, link2: 800x100, link3: 800x50
[18:54] <c3r1c3_win> grubbins2: Hmm... changing the height is what fixed it for me.
[18:55] <c3r1c3_win> grubbins2: Earlier you said you were running OBs 0.11.2, correct?
[18:55] <grubbins2> yes c3r1c3_win 0.11.2 for MAc
[18:55] <c3r1c3_win> grubbins2: Can you upgrade to 0.11.4 and try that?
[18:55] <grubbins2> sure do you have a download link? web site is still at 0.11.2
[18:56] <grubbins2> I tried this one but 404: https://github.com/jp9000/obs-studio/releases/download/0.11.4/obs-0.11.4-installer.dmg [^]
[18:57] <c3r1c3_win> grubbins2: In OBs go to help->Update... (or on a mac I think it's apple menu->Update.
[18:57] <secretply> 0.11.3 and 0.11.4 are for Windows only.
[18:57] <grubbins2> it says 0.11.2 is the latest
[18:57] <secretply> There's no .dmg files for those versions.
[18:57] <grubbins2> yeah that is what I thought RE: 0.11.4 for Mac
[18:57] <c3r1c3_win> Hmm... but thu bug fix in 0.11.4 should be universal
[18:58] <secretply> Says windows-only hotfix.
[18:58] <grubbins2> c3r1c3_win: what resolutions were you using when you got it to work?
[18:58] <c3r1c3_win> link 2 is 800x600
[18:58] <c3r1c3_win> Link 3 is 800x 350
[18:59] <c3r1c3_win> grubbins2: Can you hit the alert test button?
[18:59] <grubbins2> yes 1 sec
[18:59] <grubbins2> test follow alert
[18:59] <c3r1c3_win> Hmm... All 3 are working for me
[18:59] <c3r1c3_win> grubbins2: Try removing them and re adding them one at a time...
[19:00] <grubbins2> ok I will try that
[19:02] <grubbins2> c3r1c3_win: I removed ALL browser sources from OBS, restarted OBS, and tried to add one source: http://www.twitchalerts.com/widgets/donation-goal?token=65C6E84B2D3011756789 [^]
[19:03] <grubbins2> c3r1c3_win: it does not render
[19:03] <c3r1c3_win> grubbins2: Does that work?
[19:03] <c3r1c3_win> grubbins2: Hmm...
[19:04] <grubbins2> I used 800x600 defaults and changed the link to http://www.twitchalerts.com/widgets/donation-goal?token=65C6E84B2D3011756789 [^] but nada
[19:04] <c3r1c3_win> grubbins2: Hmm... I think you'll have to file a bug report
[19:04] <c3r1c3_win> https://obsproject.com/mantis/my_view_page.php [^]
[19:13] <grubbins2> c3r1c3_win: which browser plugin are you using with Mac 0.11.2? https://obsproject.com/forum/resources/browser-plugin.115/ [^] 1.0.10 (Experimental) OR 1.0.6 (Stable) ?
[19:14] <grubbins2> c3r1c3_win: I scorched earth to see if I can get what you have working
[19:14] <c3r1c3_win> grubbins2: I think it's the Experimental one on my mac.
[19:15] <c3r1c3_win> (I'm at my PC right now, so I can't verify that) 
```

**Comment by DDRBoxman** [comment]
I'm going to close this, it's a fairly old issue and I haven't heard similar reports or experienced this lately.
